notebooks/301-tensorflow-training-openvino/301-tensorflow-training-openvino.ipynb

```diff
@@ -37,7 +37,7 @@
     "id": "KwQtSOz0VrVX"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    train_ds = tf.keras.utils.image_dataset_from_directory(
</pre><p>is it compatible with which tf version syntax?</p><p>Is it possible situation that user have different tf version and does not have such module? (e.g. tf 2.12 does not support python3.7, so it can have lower version)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1001/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Changed to a compatible version for both keras <2.8 and for >=2.8